core/modules/layout/layout.module

```diff
@@ -1549,3 +1555,36 @@ function layout_set_handler_contexts(array &$contexts, array $all_contexts, arra
   // All required contexts populated successfully.
   return TRUE;
 }
+
+/**
+ * Gets the title description for the current layout.
+ *
+ * @param $layout
+ *   Layout object.
```

Comment:
As below, let's code hint the PHPdoc here. `@param $layout` can be `@param Layout $layout`. Even though it seems redundant to human readers, it helps with IDE completion.